include/aikido/distance/ConfigurationRanker.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef AIKIDO_DISTANCE_CONFIGURATIONRANKER_HPP_
+#define AIKIDO_DISTANCE_CONFIGURATIONRANKER_HPP_
+
+#include <dart/dynamics/dynamics.hpp>
+#include "aikido/statespace/ScopedState.hpp" // TODO: Remove this.
+#include "aikido/statespace/dart/MetaSkeletonStateSpace.hpp"
+
+namespace aikido {
+namespace distance {
+
+class ConfigurationRanker
+{
+public:
+  /// Constructor
+  ///
+  /// \param[in] metaSkeletonStateSpace Statespace of the skeleton.
+  /// \param[in] metaskeleton Metaskeleton of the robot.
+  /// \param[in] ikSolutions List of IK solutions to be ranked.
+  ConfigurationRanker(
+      statespace::dart::ConstMetaSkeletonStateSpacePtr metaSkeletonStateSpace,
+      ::dart::dynamics::ConstMetaSkeletonPtr metaSkeleton,
+      const std::vector<statespace::StateSpace::State*> ikSolutions);
```

Comment:
I don't think we need to take in the `ikSolutions` here. Maybe we'd want to reuse the ranker with a different set of `ikSolutions`?